www/src/pages/ru/usage/trpc.md

```diff
@@ -43,22 +43,27 @@ tRPC требует много шаблонного кода, который `cr
 
 ### 📄 `server/trpc/context.ts`
```

Comment:
Check the English docs. They might already have changed the file name due to change in file structure!

```suggestion
### 📄 `server/api/trpc.ts`
```

Reply:
![image](https://user-images.githubusercontent.com/23616925/212693089-324df2fb-4a8f-45bf-b258-a8b2751f822d.png)